iroh-net/src/magicsock.rs

```diff
@@ -1690,6 +1693,69 @@ impl AsyncUdpSocket for Handle {
             (_, Some(ipv6)) => Ok(*ipv6),
         }
     }
+
+    fn max_transmit_segments(&self) -> usize {
+        if let Some(pconn6) = self.pconn6.as_ref() {
+            std::cmp::max(
+                pconn6.max_transmit_segments(),
+                self.pconn4.max_transmit_segments(),
+            )
```

Comment:
Somewhere, some change I wanted to make got lost. I think in *theory* this should be `min`, not `max`.
In the sense of "this is how many transmit segments at a time the socket can handle". And if we use `max` here, we may bombard a socket with more segments that it can handle.

That said, in practice the value for `max_transmit_segments` doesn't change between IPV4 and IPV6, and it's either 512 or 1 on windows, and either 64 or 1 on unix.

```suggestion
            std::cmp::min(
                pconn6.max_transmit_segments(),
                self.pconn4.max_transmit_segments(),
            )
```